Core/ALife.Core/WorldObjects/Agents/Brains/IBrain.cs

```diff
@@ -5,5 +5,10 @@ public interface IBrain
         void ExecuteTurn();
         IBrain Clone(Agent self);
         IBrain Reproduce(Agent self);
+
+        bool CloneEquals(IBrain cloneBrain);
+
+        string ExportNewBrain();
+
```

Comment:
LINE SPACING!?!?! :P

Reply:
As discussed, this is to create separation of concerns to clearly group functionality